app/LoginModule/LTI/LTIHelper.php

```diff
@@ -115,15 +117,27 @@ private function getLoginPrefix($lti_user) {
 
     public function sendResult($lti_connection_id, $score) {
         $this->connectPDO();
-        $lc = LtiConnection::find($lti_connection_id);
-        if(!$lc) {
+        $connection = LtiConnection::find($lti_connection_id);
+        return $this->sendConnectionResult($connection);
+    }
+
+
+    public function sendResultByContent($user_id, $lti_content_id, $score) {
+        $this->connectPDO();
+        $connection = LtiConnection::where('user_id', $user_id)->where('lti_content_id', $lti_content_id)->first();
+        return $this->sendConnectionResult($connection);
```

Comment:
and you do not pass the score parameter? it is implicit?

Reply:
fixed

Reply:
How did it work before your change ? You did not tested your implementation ?

Reply:
may be I changed code after run